.github/workflows/ci.yml

```diff
@@ -118,21 +118,21 @@ jobs:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
         db-backend: ["postgresql"]
-        nautobot-version: ["1.4.0b1"]  # Change to latest or stable
+        nautobot-version: ["1.4.0", "latest"]
```

Comment:
```suggestion
        nautobot-version: ["1.4", "latest"]
```

Reply:
@itdependsnetworks do you want minimum version here & latest OR do you want any 1.4 & latest?

Reply:
Whatever the standard is.